src/gui/dialogs/migrate_version_selection.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+   Copyright (C) 2008 - 2018 by Jörg Hinrichs <joerg.hinrichs@alice-dsl.de>
```

Comment:
This line is an old cut&paste.

Reply:
I usually just delete the name and email and write "by The Battle for Wesnoth Project", essentially merging the two lines. Up to you though.